torchtext/experimental/datasets/language_modeling.py

```diff
@@ -145,23 +141,23 @@ def WikiText103(tokenizer=None, root='.data', vocab=None, data_select=('train',
             just a string 'train'. If 'train' is not in the tuple or string, a vocab
             object should be provided which will be used to process valid and/or test
             data.
-        single_line: whether to return all tokens in a single line.
-            (Default: True)
-            By default, all lines in raw text file are concatenated into a single line.
-            Use `single_line = False` if one wants to get data line by line.
 
     Examples:
         >>> from torchtext.experimental.datasets import WikiText103
         >>> from torchtext.data.utils import get_tokenizer
         >>> tokenizer = get_tokenizer("spacy")
         >>> train_dataset, valid_dataset, test_dataset = WikiText103(tokenizer=tokenizer)
+        >>> # convert dataset into a single line
```

Comment:
I don't think this is necessary right here. We could however include this in one of our tutorials to show how easy it can be.